test/integration/features/basic.feature

```diff
@@ -21,6 +21,17 @@ Feature: Basic
      sudoer
      """
 
+  Scenario: Current Context should be minishift
```

Comment:
Nice. The only thing I am wondering about is whether we should extend on the testing. Right now you only check whether initially the context is _minishift_.

We could extend this to test the whole flow. After verifying that the current context is _minishift_ and maybe that the current project (oc project -q) is myproject, we write a new scenario "User can switch context". 

In this scenario we create a dummy context `oc config set-context fubar ` and switch to it `oc config use-context fubar`. Now a `oc project -q` should return an error.

Then in the final scenario "User can switch back to minishift context", we do a `oc config use-context minishift` followed by `oc project myproject` which should now work again, right? WDYT?